src/components/Tree/Tree.cy.tsx

```diff
@@ -7,7 +7,7 @@ import { treeNodesMock } from '@components/Tree/utils/mocks';
 
 const TreeComponent = ({ onSelect }: { onSelect?: (id: Nullable<string>) => void }) => {
     return (
-        <Tree id="treeId" onSelect={onSelect}>
+        <Tree id="treeId" onSelect={onSelect} baseItemPadding={{ x: 50, y: 20 }}>
```

Comment:
As we are the only one to use the tree for now, we could set it as default wdyt?
Designers should align that, having different paddings on tree is weird.

Reply:
Already aligned, custom is the way forwards since it's used in Analytics